updater.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/bash
```

Comment:
If you haven't already: following [this](https://github.com/hyperledger/aries-cloudagent-python/blob/main/scripts/run_docker) example might help having a cross-OS compatibile script.